rfcs/0085-ZHF-on-master-and-freeze-dates.md

```diff
@@ -0,0 +1,104 @@
+---
+feature: nixos-freeze-dates
+start-date: 2021-01-17
+author: Jonathan Ringer (@jonringer)
+co-authors:
+related-issues: [NixOS release schedule](https://github.com/NixOS/rfcs/pull/80)
+---
+
+# Summary
+[summary]: #summary
+
+To further minimize the amount of risk going into a NixOS release, I propose
+to set freeze dates regarding changes to critical packages being merged to
+staging. In addition, I also propose to stabilize the release on master
+instead of the release branch, thus avoiding the need to backport every pull
+request, and take advantage of the stabilization that unstable normally
+receives.
```

Comment:
This should also specify what happens with new packages/modules.

Reply:
New packages are welcome. Changes to modules are welcome, assuming it doesn't cause regressions, and the module changes will probably need to have documentation so it's not additional work trying to curate the release notes.

I will modify the definitions section to say that "freezing" will still allow new package and module additions.